src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {
+    private String color;
+    private String figure;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setFigure(String figure) {
+        this.figure = figure;
+    }
+
+    public String getFigure() {
+        return figure;
+    }
+
+    public abstract void draw();
```

Comment:
move this method to interface Drawable and let Figure to implement this interface